src/auth.js

```diff
@@ -0,0 +1,16 @@
+import {get,readable} from 'svelte/store';
+var value;
+export const state =  readable(value,function auth(set){
+fetch("https://auth.canadiana.ca/1/auth",{
+credentials:"include",
+})
+.then(function(response, data){
+    if(response.status == 200){
+       set(value = response.status);
```

Comment:
I don't think you need to set `value` within the call to `set` here.

```suggestion
       set(response.status);
```